packages/crypto-server/src/@types/types.ts

```diff
@@ -4,16 +4,16 @@
 };
 
 export type IHeadersGenerate = {
-  type: string;
-  bits: string;
-  name: string;
+  curve: any;
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/sebastienrousseau/crypto-service/pullRequest?prid=9768072)
```suggestion
  curve: unknown;
```